discordrp_mpris/__main__.py

```diff
@@ -124,7 +125,11 @@ async def tick(self) -> None:
             details_fmt = "{title}\nby {artist}"
         else:
             details_fmt = "{title}"
-        activity['details'] = self.format_details(details_fmt, replacements)
+        activity['details'] = shorten(
+            self.format_details(details_fmt, replacements),
+            width=128,
```

Comment:
Please extract this to a constant (ideally with an acompanying comment), so that it is more clear how it was chosen.

Reply:
I added the constant `DETAILS_BYTES_MAX`